### PR TITLE
Remove print in test

### PR DESCRIPTION
**Pull request by carllin**
#### Problem
Println crept into PR 🗡

#### Summary of Changes
Remove

Fixes #

### CODE REVIEW

**Comment by mvines** [comment]
image

**Comment by mvines** [comment]
Can we welcome `ledger/` into this elite group?
https://github.com/solana-labs/solana/blob/08ba27627d1d8eaede0691e3885135896a09865c/ci/nits.sh#L20-L31

**Comment by solana-grimes** [comment]
:scream: New commits were pushed while the automerge label was present.

**Comment by solana-grimes** [comment]
:broken_heart: Unable to automerge due to CI failure

File: ledger/src/blocktree.rs [hunk 1]
```diff
@@ -2127,8 +2127,8 @@ pub fn create_new_ledger(ledger_path: &Path, genesis_config: &GenesisConfig) ->
     if !output.status.success() {
         use std::io::{Error as IOError, ErrorKind};
         use std::str::from_utf8;
-        eprintln!("tar stdout: {}", from_utf8(&output.stdout).unwrap_or("?"));
-        eprintln!("tar stderr: {}", from_utf8(&output.stderr).unwrap_or("?"));
+        error!("tar stdout: {}", from_utf8(&output.stdout).unwrap_or("?"));
```
Comment (mvines) [inline ledger/src/blocktree.rs]: ha!  we smoked a couple more out.  excellent

**Pull request merged**